README.md

```diff
@@ -64,13 +64,20 @@ def step(carry, x):
 last_number, all_numbers = lax.scan(step, 0, jnp.arange(n))
 ```
 
-will update every-other steps.
+will update every other step.
 
 ## Why JAX-tqdm?
 
-JAX functions are [purely functional](https://jax.readthedocs.io/en/latest/notebooks/Common_Gotchas_in_JAX.html#pure-functions), so side effects such as printing progress when running scans and loops are not allowed. However, the [host_callback module](https://jax.readthedocs.io/en/latest/jax.experimental.host_callback.html) has primitives for calling Python functions on the host from JAX code. This can be used to update a Python tqdm progress bar regularly during the computation. JAX-tqdm implements this for JAX scans and loops and is used by simply adding a decorator to the body of your update function.
```

Comment:
moar nitpick (sorry :) ): _JAX functions are pure_ (not purely)

Reply:
(I mean to add this comment in the edited code by the way)

Reply:
Jeez sorry, duh! Will fix!

Reply:
Fixed, slow brain on a Monday morning!